cmd/podman/common/build.go

```diff
@@ -585,7 +585,7 @@ func buildFlagsWrapperToOptions(c *cobra.Command, contextDir string, flags *Buil
 	}
 
 	if flags.IgnoreFile != "" {
-		excludes, err := parseDockerignore(flags.IgnoreFile)
+		excludes, _, err := parse.ContainerIgnoreFile(contextDir, flags.IgnoreFile, containerFiles)
```

Comment:
The function comment says
> Deprecated since this might become internal only, please avoid relying on this function.

So what is the status with that, either we should not use it or change the buildah comment to actual commit to that as API? Personally I am all for such code sharing.
cc @nalind

Reply:
Right I agree. Just pointing out that the buildah comment should then get updated if we start to depend on it here.